build/serve.js

```diff
@@ -120,7 +120,7 @@ gulp.task('serve:prod', ['build-frontend'], function () {
 gulp.task('spawn-backend', ['backend', 'kill-backend'], function () {
   runningBackendProcess = child.spawn(
       path.join(conf.paths.serve, conf.backend.binaryName),
-      [`--port=${conf.backend.devServerPort}`], {stdio: 'inherit'});
+      [`--apiserver-host=${conf.backend.apiServerHost}`,`--port=${conf.backend.devServerPort}`], {stdio: 'inherit'});
```

Comment:
Style nitpicks: please limit width to 100 chars and have spaces after `,`. Also, rebase the commit because I've already modified this line.